jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/HBaseIntegrationTest.java

```diff
@@ -0,0 +1,441 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxscraper.target_systems;
+
+import static io.opentelemetry.contrib.jmxscraper.target_systems.MetricAssertions.assertGauge;
+import static io.opentelemetry.contrib.jmxscraper.target_systems.MetricAssertions.assertGaugeWithAttributes;
+import static io.opentelemetry.contrib.jmxscraper.target_systems.MetricAssertions.assertSum;
+import static io.opentelemetry.contrib.jmxscraper.target_systems.MetricAssertions.assertSumWithAttributes;
+import static org.assertj.core.data.MapEntry.entry;
+
+import io.opentelemetry.contrib.jmxscraper.JmxScraperContainer;
+import java.nio.file.Path;
+import java.time.Duration;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+public class HBaseIntegrationTest extends TargetSystemIntegrationTest {
+  private static final int DEFAULT_MASTER_SERVICE_PORT = 16000;
+
+  @Override
+  protected GenericContainer<?> createTargetContainer(int jmxPort) {
+    return new GenericContainer<>("dajobe/hbase")
+        .withEnv("JAVA_HOME", "/usr/lib/jvm/java-8-openjdk-amd64")
+        .withEnv("HBASE_OPTS", "-XX:+UseConcMarkSweepGC")
```

Comment:
[minor] Is the GC algorithm is really needed for the JMX metrics ? If yes then adding a comment to explain why could be relevant, otherwise we can simply remove it as we are not really doing a production ready setup here.

Reply:
Removed